.github/workflows/ExportOptions.plist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I feel like this file doesn't belong to this `workflows/`, it's an iOS build config file so should it locate in `ios/` or a new dir from the root somehow?

Reply:
@sleepylee I moved it into `ios/config`